.nais/nais-dev.yaml

```diff
@@ -36,7 +36,10 @@ spec:
       value: >
         {
           "cors": {
-            "origin": "https://veilarbpersonflate.intern.dev.nav.no"
+            "origin": [
+              "https://veilarbpersonflate.intern.dev.nav.no",
+              "https://arbeidssokerregistrering.intern.dev.nav.no"
```

Comment:
Det går ikke ann å si at dersom det ender på intern.dev.nav.no så skal det være greit? (altså mer generelt) Er kanskje for lite sikkert? 🤔  eller ikke mulig 🤷‍♀️

Reply:
Det er sikkert ikke noe i vegen for å bruk en mer generell regel for subdomenet `*.intern.dev.nav.no`. Samtidig så er det kanskje greit å holde det spesifikt og samtidig praktisere "principle of least privilege"?